platform/default/src/mbgl/storage/offline_database.cpp

```diff
@@ -76,30 +76,36 @@ void OfflineDatabase::changePath(const std::string& path_) {
     initialize();
 }
 
-void OfflineDatabase::cleanup() {
+void OfflineDatabase::cleanup(bool deleteDBFile) {
```

Comment:
These statements cannot throw (I can't see how):
```c++
statements.clear();
db.reset();
```
`util::deleteFile(path);` this can throw IOException when file is deleted, thus, I think last catch is unreachable. Also, error should be something like "cannot delete database file".

Reply:
You are right, clearing DB doesn't throw. Fixed.